File: src/main/java/com/refah/walletwrapper/WalletWrapperApplication.java
package com.refah.walletwrapper;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@SpringBootApplication
@EnableJpaRepositories(basePackages = "com.refah.walletwrapper.repository")
public class WalletWrapperApplication {

    public static void main(String[] args) {
        SpringApplication.run(WalletWrapperApplication.class, args);
    }
}

File: src/main/kotlin/com/refah/walletwrapper/WalletWrapperApplication.kt
package com.refah.walletwrapper

import com.cosium.spring.data.jpa.entity.graph.repository.support.EntityGraphJpaRepositoryFactoryBean
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaRepositories


@SpringBootApplication
@EnableJpaRepositories(
    basePackages = ["com.refah.walletwrapper.repository"],
    repositoryFactoryBeanClass = EntityGraphJpaRepositoryFactoryBean::class
)
class WalletWrapperApplication

fun main(args: Array<String>) {
    runApplication<WalletWrapperApplication>(*args)
}
